index.js

```diff
@@ -1,69 +1,100 @@
 // ***** Question 1 *****
+
+let counter = 1
+counter = 2
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
+console.log("%cQuestion 1", "color: red") 
 
-// console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
+console.log(counter) 
+// => 2
+console.log("%c----------", "color: red") 
 
 
 // ***** Question 2 *****
+const name = "Raffy"
 // *** Uncomment the lines below to test
 // *** after testing, comment the line causing the error back in 
 // *** otherwise, the error will stop the rest of your code from running
-// console.log("%cQuestion 2", "color: red") 
-
-// console.log(name)
+console.log("%cQuestion 2", "color: red") 
+console.log(name)
 // => "Raffy"
 // name = "Not Raffy" 
 // => TypeError
-// console.log("%c----------", "color: red") 
+console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 3 *****
+function drinkWater(currentThirstLevel){
+  console.log("Man I sure am thirsty")
+  currentThirstLevel -= 1
+  console.log ("Ahh that hits the spot")
+  return currentThirstLevel
+}
+
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 3", "color: red") 
+console.log("%cQuestion 3", "color: red") 
 
-// console.log(drinkWater(12))
+console.log(drinkWater(12))
 // // => "Man I sure am thirsty"
 // // => "Ahh that hits the spot"
 // // => 11
-// console.log("%c----------", "color: red") 
+console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 4 *****
+
+function sameSameButDifferent(num, maybeNum){
+  if (num === maybeNum)
+    return "same same"
+  else if (num == maybeNum)
+    return "same same (but different)"
+  else
+    return "different"
+}
+
```

Comment:
This will work, but just a couple notes:

- for if/else statements, it's better to use curly parentheses to separate out each condition for readability/flexibility:

  `if (num === maybeNum) {
    return "same same"
  } else if (num == maybeNum) {
    return "same same (but different)"
  } else {
    return "different"
  }`
- Also, you could also convert the string to a number with parseInt to avoid ==:
`} else if (num === parseInt(maybeNum)) {`